### ISSUE TITLE
Can't create instance of RichWebView 

### ISSUE THREAD

**Issue by lemmingDev** [myflashlab/webView-ANE#227]
## Environment ##
* Android Version:  targetSdkVersion="26"
* iOS Version: NA
* AIR Version: 30 and 32 Beta
* Adobe Animate 2019

## Problem Description ##
Can't create instance of RichWebView
Project compiles, but I get this error:
ArgumentError: Error #1063: Argument count mismatch on com.myflashlab.air.extensions.dependency::OverrideAir$/applyToAneLab(). Expected 1, got 0.

when it tries to load the following line of code: ```var _ex:RichWebView = new RichWebView(this.stage);

```xml
```My XML file is set up correctly
```actionscript
var _ex:RichWebView = new RichWebView(this.stage);

## Steps to Reproduce ##
Basically just running the example code

## Known Workarounds ##
